json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -364,6 +366,9 @@ protected function render_response_key( $key, &$response, $is_user_logged_in ) {
 			case 'quota' :
 				$response[ $key ] = $this->site->get_quota();
 				break;
+			case 'launch_status' : 
+				$response[ $key ] = $this->site->get_launch_status();
```

Comment:
I believe you may have to add `get_launch_status()` here as well:
https://github.com/Automattic/jetpack/blob/e9c0e0c1a02c07afed00e2c9d47d9b70184e6360/sal/class.json-api-site-base.php